modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -126,7 +128,25 @@ instead.
 `CountBadge` now uses `createComponent`, Canvas Tokens, and our new styling utilities. The component
 API has not been changed, and it should behave identically as it did in previous versions.
 
-### FormField Preview
+### Text
+
+`Text` has been refactored to use our new styling utilities and new tokens and no longer
+extends `Box` component. The component API has not been changed, and it should behave identically as
+it did in previous versions and support Emotion `styled` and style props. Type level components,
+`Title`, `Heading`, `BodyText` and `Subtext`, have not been changed, but because they extend `Text`
+components, all changes made for `Text` will be applied for them as well.
```

Comment:
```suggestion
`Text` has been refactored to use our new styling utilities, new tokens and no longer
extends the `Box` component. The component API has not been changed and it should behave identically as
it did in previous versions while still supporting Emotion `styled` and style props. Type level components: `Title`, `Heading`, `BodyText` and `Subtext`, have not been changed because they extend `Text` components and all changes made to `Text` will be applied to them as well.
```